sys/include/net/dhcpv6/client.h

```diff
@@ -170,6 +199,46 @@ void dhcpv6_client_conf_prefix(unsigned netif, const ipv6_addr_t *pfx,
                                unsigned pfx_len, uint32_t valid,
                                uint32_t pref);
 
+/**
+ * @brief   Checks if the given network interface is configured
+ *          to use DHCPv6 IA_NA
+ *
+ * @param[in] netif     Network interface to check.
+ *
+ * @return  true, if the network interface is set up for IA_NA.
+ */
+bool dhcpv6_client_check_ia_na(unsigned netif);
+
+/**
+ * @brief   Configures a address lease that is provided by the server.
+ *
+ * @param[in] netif     Network interface the address was for.
+ * @param[in] addr      The assigned address.
+ *
+ * @return  true, if the address was configured successfully.
```

Comment:
Why not keep the diagnostic error return code?

Reply:
I changed the return type to `void` now in 6941fc305ce7c5a99cf8205f437d5f4ba2eb9ed9

Reply:
The return value from `gnrc_netif_ipv6_addr_add` (or another network stack's equivalent) is now being passed over (see 49e3ec411ba49caf5dc6f079db434f1b271181d7). I wasn't sure if the error codes from `gnrc_netif_ipv6_addr_add` should also be added to the documentation of `dhcpv6_client_add_addr` as I thought that these could be network stack specific.

Reply:
> should also be added to the documentation of dhcpv6_client_add_addr as I thought that these could be network stack specific.

Well at least you can add to the documentation that the on error, the function shall return a negative value.